lib/src/tasks/format/api.dart

```diff
@@ -81,34 +86,38 @@ FormatTask format({
   return task;
 }
 
-/// Returns a set of files/directories within [directories] to be formatted,
-/// with [exclude] excluded.
+/// Returns a set of files within [paths] to be formatted, with [exclude] excluded.
+///
+/// [paths] can contain both files and directories. Files within directories
+/// will be processed recursively, ignoring symlinks.
 ///
-/// If [exclude] is not empty, then [directories] will be expanded recursively
-/// (ignoring symlinks) to all of its files. Otherwise, it will not be expanded.
+/// If [exclude] is not empty, the return value will include [paths], expanded
+/// to all matching files. Otherwise, it will not be expanded.
 ///
-/// To force expansion of [directories], set [alwaysExpand] to `true`.
+/// To force expansion, set [alwaysExpand] to `true`.
 FilesToFormat getFilesToFormat({
-  List<String> directories: defaultDirectories,
+  List<String> paths: defaultPaths,
```

Comment:
Not a breaking change because the PR that adds this function hasn't been merged yet